main.go

```diff
@@ -132,6 +135,58 @@ func sendMetric(v []string, t map[string]string, message string, tags [][]string
 	fmt.Fprintf(conn, put)
 }
 
+func rejectMetric(app string, metric string, metricType string) {
+	appParts := strings.SplitN(app, "-", 2)
+	appName := appParts[0]
+	appSpace := appParts[1]
+
+	rejectMessage := struct {
+		Log        string      `json:"log"`
+		Stream     string      `json:"stream"`
+		Time       time.Time   `json:"time"`
+		Kubernetes interface{} `json:"kubernetes"`
+		Topic      string      `json:"topic"`
+	}{
+		"Unique metrics limit exceeded. Metric discarded: [" + metricType + "] " + metric,
+		"stdout",
+		time.Now(),
+		struct {
+			PodName       string `json:"pod_name"`
+			ContainerName string `json:"container_name"`
+		}{
+			"akkeris-warning",
+			appName,
```

Comment:
can we make the pod name something that could never be used as a pod name so that its not confusing in the routed logs ( i know, i know ).  Usually we use forward slashes for http logs (`akkeris/router`) and for events (`akkeris/events`), can this be changed to be (`akkeris/warning`) or (`akkeris/anomaly`)?

Reply:
Read you loud and clear - how about `akkeris/metrics`

Reply:
Changed to `akkeris/metrics`